docs/blog/connected-registry.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Connected Registry Private Preview
+description: Private preview for ACR connected registry feature.
+ms.topic: post
+ms.date: 01/05/2021
+ms.author: memladen
+author: toddysm
+ms.custom:
+---
+
+# Private Preview - ACR Connected Registry Feature
+We are announcing the private preview of Azure Container Registry (ACR) connected registry feature.
+
+The connected registry feature of ACR allows you to deploy a registry on your premises and synchronize images between the ACR and your premises. It brings the container images and OCI artifacts closer to your on-premise container workloads and increases their performance and reliability.
```

Comment:
Suggest "...synchronize images between **an Azure container registry in the cloud** and your premises..."

Not essential, but an image could also help here.